media/css/pebbles/components/masthead.less

```diff
@@ -0,0 +1,121 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import (reference) '../lib.less';
+
+
+#masthead .content {
+    padding-top: 0;
+}
+
+.masthead-logo {
+    .font-size(1rem);
```

Comment:
Again, we should stick to one standard unit for font-size declarations? It's `rem` here but `px` below.